packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -82,20 +82,13 @@ export default class VaDropdown extends mixins(DropdownPropsMixin) {
 
   get triggeredValue (): boolean {
     const getTriggeredValue = () => {
-      switch (this.trigger) {
-      case 'hover':
-        return this.isMouseHovered
-      case 'click':
-        return this.isClicked
-      case 'none':
-        return this.modelValue
-      default:
-        return false
+      if (['click', 'hover', 'none'].includes(this.trigger)) {
+        return this.valueComputed
       }
+      return false
     }
 
     const triggeredValue = getTriggeredValue()
-
     this.$emit('trigger', triggeredValue)
```

Comment:
Hmm. That's not how getters work.
There should never be the case where we emit from getter.